packages/crud/src/vaadin-crud.js

```diff
@@ -436,6 +484,7 @@ class Crud extends SlotMixin(ElementMixin(ThemableMixin(PolymerElement))) {
        */
       editorOpened: {
         type: Boolean,
+        reflectToAttribute: true,
```

Comment:
nit: would be nice to avoid having the host attributes `editor-opened` and `editor-position` but that would probably require exposing the `#container` as a `part` :/

Reply:
I don't have a strong opinion. @jouni What would you recommend to do in this case?

Reply:
Removing `editor-position` host attribute could be considered a breaking change even though it's intended for internal styling

Reply:
I don't remember the implementation that well. But from a user perspective, I would like to have the option to style the component(s) based on these two states as well. In general, I agree that we should avoid unnecessary host attributes, but at first glance these seem okay to me.

Reply:
Btw, is there a need to scope the selectors by `[editor-opened]` in the first place. I didn't notice a difference without it